tutorials/1_basic_square/README.md

```diff
@@ -0,0 +1,153 @@
+# Basic: Rendering a simple square on the screen
+
+This tutorial will introduce you to:
+
+ - `Game`: The basic class to build a game on, you will understand its structure and how to use it to build a simple game
+ - `GameWidget`: The widget used to place your game on your Flutter widget tree
+ - Basic rendering: You will get introduced to the basics of using the `Canvas` class from `dart:ui` which is the class that Flame uses for rendering your game.
+
+All the code of this tutorial code can be found [here](./code) and is based on the Flame `1.0.0-rc8` version.
+
+By the end of this tutorial you will have built a simple game that renders a square bouncing on the screen that will look like this:
+
+![Preview](./media/preview.gif)
+
+## Building your first game
+
+`Game` is the most basic class that you can use to build your game, it includes the necessary methods for creating a basic game loop and methods for the lifecycle of a Flame game.
+
+For more complex games, you will probably want to use `BaseGame`, which has a lot of utilities that will make your life easier on more complex game, we will cover that on later tutorials, for this one the `Game` class will be enough for the concepts that you will learn here.
+
+`GameWidget` is, like its name suggest, a Flutter widget that will run your game and place it inside the Flutter widget tree.
+
+As a first step, lets just get a Game instance running, for that we will need to create our own class that extends Flame's Game class, implement its methods, and pass an instance of that Game to a GameWidget. Something like this:
+
+```dart
+import 'package:flutter/material.dart';
+import 'package:flame/game.dart';
+
+void main() {
+  final myGame = MyGame();
+  runApp(
+    GameWidget(
+      game: myGame,
+    ),
+  );
+}
+
+class MyGame extends Game {
+  @override
+  void update(double dt) { /* TODO */ }
+ }
```

Comment:
@erickzanardo please remove this line number 41 as it is incorrect. thanks!

Reply:
Fixed, took me longer than I am comfortable to accept to realize that this `}` was the problem.